cli/src/commands/login.ts

```diff
@@ -3,8 +3,7 @@ import * as chalk from 'chalk';
 import * as firebase from 'firebase';
 import * as open from 'open';
 import { refBuilder } from '../firebase/fb';
-import { Observable } from 'rxjs/Observable';
-import { fromPromise } from 'rxjs/observable/fromPromise';
+import { Observable, from as fromPromise } from 'rxjs';
```

Comment:
At the top, we used `from` instead of `fromPromise`. Maybe we should be consistent and rename all the `fromPromise` calls to `from`?

Reply:
Good catch! I wonder if that's a bug in rxjs-tslint... Will fix this!